packages/jest-runner/package.json

```diff
@@ -13,6 +13,7 @@
     "@jest/console": "^24.7.1",
     "@jest/environment": "^24.8.0",
     "@jest/test-result": "^24.8.0",
+    "@jest/transform": "^24.8.0",
```

Comment:
you need to add it to `references` in `tsconfig` as well so `tsc` builds things in the correct order

Reply:
done

Reply:
done